geometry_msgs/Point[] points_center
geometry_msgs/Point[] points_up
geometry_msgs/Point[] points_down
geometry_msgs/Point[] points_up_up
geometry_msgs/Point[] points_down_down
uint8 flag
